js/index.js

```diff
@@ -1,7 +1,114 @@
+
+//this works if I remove the paragraphs, as I am counting white spaces for words. not sure how to catch the 2 words after a paragraph break.
+var wordCount = 0;
+var searchTerm = ` et `;
+var etCount = 0;
+var nextIndexOfTerm = lorem.indexOf(searchTerm); //first occurence of search term. returns -1 if none found.
+
+for (var i = 0; i < lorem.length; i++) {
+  if (lorem[i] === ` `) {
+    wordCount++;
+  }
+}
+wordCount++; //this is for the last word of the string which is not succeeded by a space.
+console.log(`Number of words in string: ${wordCount}`);
+
+//for the length of string, and if indexOf finds search term, count++ and update nextindex for search.
+for (var i = 0; i < lorem.length && nextIndexOfTerm !== -1; i++) {
+  etCount++;
+  nextIndexOfTerm = lorem.indexOf(searchTerm, nextIndexOfTerm + 1);
+}
+console.log(`Number of occurences of "${searchTerm}": ${etCount}`);
+
+//Bonus 2
+
+var phraseToCheck = `Amor, Roma`;
+var newPhrase = ``;
+var newReversePhrase = "";
+
+for (var i = 0; i < phraseToCheck.length; i++) {
+  if (phraseToCheck[i] === " ") {
+    //this takes care of spaces, but symbols like ! or ? arent taken care of here. didnt want to have a bunch of checks for symbols.
+    continue;
+  } else {
+    newPhrase += phraseToCheck[i].toLowerCase();
+  }
+}
+console.log(`Initial Phrase: ${newPhrase}`);
+
+//same construction as above but for reverse Phrase
+for (var i = phraseToCheck.length - 1; i >= 0; i--) {
+  if (phraseToCheck[i] === " ") {
+    continue;
+  } else {
+    newReversePhrase += phraseToCheck[i].toLowerCase();
+  }
+}
+console.log(`Reverse phrase: ${newReversePhrase}`);
```

Comment:
Here instead of doing all the checks again, you could reverse the "newPhrase" that you just cleaned from spaces, saves you some code ;)